clustering/common/src/main/java/org/jboss/as/clustering/controller/AddStepHandler.java

```diff
@@ -90,11 +63,31 @@ protected void rollbackRuntime(OperationContext context, ModelNode operation, Re
     }
 
     @Override
+    protected void populateModel(final ModelNode operation, final ModelNode model) throws OperationFailedException {
+        for (AttributeDefinition attribute : this.descriptor.getAttributes()) {
+            attribute.validateAndSet(operation, model);
+        }
```

Comment:
What I've called "extra parameters" are parameters to the resource add operation that are *not* attributes of the resource - therefore these are explicitly not populated into the resource model.